run_sim.sh

```diff
@@ -0,0 +1,33 @@
+#!/bin/bash
+
+go build -linkshared
+
+for kb in 128 256 512 1024 2048 4096 8192; do
+  for announce in 0 7 8; do
+      result=$((kb * 1024))
+      filename=shadow-$kb-$announce-1
+      python3 network_graph.py 100 70 $result $num_msgs 8 $announce
```

Comment:
Why 70? I think we had 35.

Reply:
yes, true. I think I left it in there by mistake. I'll change it.